lib/logger.js

```diff
@@ -96,8 +96,8 @@ logger.log = function (...args) {
 logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
-		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan) + '\n');
+		logger.log('Titanium CLI v' + info.version + (logger.activeSdk ? ', SDK v' + logger.activeSdk : '') + ', https://titaniumsdk.com'.cyan + '\n' + info.about.copyright);
```

Comment:
The comma before the titaniumsdk.com link is cyan instead of the default.

Again, I'm currently against showing the copyright unless we absolutely have to. In which case, I don't get why we can't just say the year: `Copyright TiDev, Inc. 2022`. I don't know if we need to say "All Rights Reserved". What do other CLI's do?

Reply:
Understood. No need for copyright. It's clean now.

<img width="1361" alt="Bildschirmfoto 2022-05-19 um 07 42 05" src="https://user-images.githubusercontent.com/1126641/169218663-5de116c4-b48c-469e-a4ed-bbcf6370ee0b.png">

Reply:
`npm` e.g. shows nothing than the help
```
sebastianklaus$ npm
npm <command>

Usage:
```

And `alloy` does like npm